Sprint-2/5-stretch-extend/format-time.js

```diff
@@ -23,3 +23,31 @@ console.assert(
   currentOutput2 === targetOutput2,
   `current output: ${currentOutput2}, target output: ${targetOutput2}`
 );
+
+const currentOutput3 = formatAs12HourClock("00:00");
+const targetOutput3 = "12:00 am";
+console.assert(
+  currentOutput3 === targetOutput3,
+  `Test failed :00:00 current output: ${currentOutput3}, target output: ${targetOutput3}`
+);
+const currentOutput4 = formatAs12HourClock("24:00");
+const targetOutput4 = "12:00 pm";
+console.assert(
+  currentOutput4 === targetOutput4,
+  `current output: ${currentOutput4}, target output: ${targetOutput4}`
+);
+ 
+const currentOutput5 = formatAs12HourClock("12:00");
+const targetOutput5 = "12:00 pm";
+console.assert(
+  currentOutput5 === targetOutput5,
+  `Test failed :12:00 current output: ${currentOutput5}, target output: ${targetOutput5}`
+);
+const currentOutput6 = formatAs12HourClock("13:50");
+const targetOutput6 = "01:50 pm";
+console.assert(
+  currentOutput6 === targetOutput6,
+  `Test failed :13:50 current output: ${currentOutput6}, target output: ${targetOutput6}`
```

Comment:
This assertion is failing for me. What do we have to change in `formatAs12HourClock` to fix it?